pom.xml

```diff
@@ -191,6 +192,13 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
```

Comment:
I assume this section is needed only for pubsub, if yes please move to pubsub module

Reply:
GCP Library Based on spring cloud 2020 has been migrated to https://github.com/GoogleCloudPlatform/spring-cloud-gcp, So I also upgraded the PubSub dependency

Reply:
If you have to change it according to your suggestion, I will deal with it later

Reply:
just not to keep common pom flooded, if this managements is used only in one sub-modules, better to move there, if 2+ sub-modules requires it, im okay to keep it here

Reply:
Thank you for your advice. I've dealt with it